### ISSUE TITLE
 Sitewide Content and IA from CTO Health Team for MHV on VA.gov Authenticated Landing Page

### ISSUE THREAD

**Issue by batemapf** [department-of-veterans-affairs/va.gov-team#53568]
## What does your team need support for? Check all that apply.
- [x] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [ ] Something else 
 
Launching the MHV on VA.gov authenticated landing page. Signed in users will access this page by clicking the `My Health` (soon to be updated to `My HealtheVet` link) in the main navigation. From there, users will be able to access MHV tools. At the outset, the majority of these tools will be hosted on the legacy MHV Liferay portal (myhealth.va.gov). As those tools migrate to VA.gov, users will begin to access them directly on VA.gov.

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [x] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
  
Early summer 2023.
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- Link to product outline:
- Link to or attach user flows: **At MVP launch...**
  - VA.gov --> sign on --> My Health in main nav --> Landing page --> Deep link to MHV legacy tool / content (excluding appointment related things)
  - VA.gov --> sign on --> My Health in main nav --> Landing page --> VA Online Scheduling
- Link to prototype or mockups:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/49026#issuecomment-1423090950
- Link to any epics/issues that may be helpful:

## Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: 3/17 (to limited % of users)
- Estimated staging review date: 3/1
- Content and IA work needed by: 3/1

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: 
-	OCTO-DE product owner: Patrick Bateman
-	Product manager: Marci McGuire
-	Designer: Ryan Thurlwell
-	FE engineer: Daniel Cloud
-	Product/team Slack channel: [mhv-on-vagov-portal-home](https://dsva.slack.com/archives/C04GDQSHD1R)

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by mnorthuis** [comment]
Attached is the spreadsheet documenting the links, groupings, order, and labels for the MVP landing page - please utilize the tab labeled "Proposed MVP features" for this information.

Highlighted in the spreadsheet are some updates to labels - these will not be implemented for design intent, but should be implemented for launch. 

Bringing @kristinoletmuskat and @laurwill into the loop on this work. 

Next steps will be to identify changes needed as each tool moves through the launch phases.  These changes will include labeling changes, add/remove of links, potential group changes, destination changes, etc.

[MHV-landing-page.xls](https://app.zenhub.com/files/133843125/0da846fc-31bf-4961-9da3-a43bdae32bc5/download)

**Comment by DanielleThierryUSDSVA** [comment]
Hi @batemapf! I worked with Mikki on the link labeling so most of my content changes are already reflected in the updated spreadsheet that she shared.

I had one more content comment to share. I don't know that the existing intro sentence quite works because it says this is "one place" to manage health care and that's not technically true at this stage.

I'd recommend language like this instead:

Welcome to My HealtheVet on VA.gov—where you can manage your VA health care and your health.

Happy to talk thru it more too if you and Ryan would like. Thanks!

**Comment by batemapf** [comment]
@DanielleThierryUSDSVA used your suggested content. how does this look? note that the `My Health` link in the main nav will be updated to say `My HealtheVet` (per this [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/23462)) so that main nav link text and the H1 of this page will match (and both say `My HealtheVet`).

<img width="1070" alt="Screenshot 2023-02-24 at 1 06 41 PM" src="https://user-images.githubusercontent.com/7645362/221269790-d29f1f90-a693-4f12-ba08-3d4055abd079.png">

**Comment by DanielleThierryUSDSVA** [comment]
@batemapf Looks good! Super minor thing: Delete spaces around the em dash and the em dash looks like it might be a little long (but that might just be me). Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @megzehn @RLHecht. The team (cc @batemapf) is aiming to launch the authenticated MHV > VA.gov landing page tomorrow. Would you be able to do a final errors-and-omissions proof on this? We're looking for typos (you'll see in the screenshot that there's an instance of a lowercase "v" in "Veteran" that the team is fixing now) and alignment with the link label spreadsheet that Mikki linked to in the earlier comments.

![image.png](https://images.zenhubusercontent.com/5e8b13016a4378a604f5e799/7dca7092-7899-4f5c-b16f-6a0692f5bf8e)

**Comment by megzehn** [comment]
Hello! 

Here are a few copy edits:
- Lowercase "s" in **In the spotlight** to keep all the headers in sentence case.
- Use sentence case for headlines. Non-branded terms should be lowercase, like the rest of the landing page (specific terms below). Branded terms, like "Veterans Health Library" and "Healthy Living Centers," can stay capitalized.
- Use curly quotes (like in VA's Whole Health Living) for consistency with our site.

Specific changes for the second recommendation:
- Lowercase "care" in Community care
- Lowercase "your" and "cholesterol" under spotlight
- Lowercase "new toxic exposure screening" under spotlight
- Lowercase "exercises for people over 50" under spotlight

**Comment by megzehn** [comment]
^ @batemapf  let me know if there are any concerns with these edits! 

CC @DanielleThierryUSDSVA and @RLHecht

**Comment by batemapf** [comment]
@megzehn thank you! could you provide a little more detail on your third bullet re: curly quotes? is the recommendation to change:

`How to use VA Video Connect`

to

`How to use "VA Video Connect"`

?

if so, in the situation where there the whole list item is a brand name (e.g., Veterans Health Library), do you recommend:

`"Veterans Health Library"`

or

`Veterans Health Library`

?

**Comment by batemapf** [comment]
👋 hey folks — i _think_ this is OK to close? or do we leave it open for a while for future iterations of the landing page?
